argoverse/evaluation/eval_tracking.py

```diff
@@ -206,8 +200,14 @@ def eval_tracks(
                         gt_data[i]["center"]["z"],
                     ]
                 )
+
                 if bbox[3] > 0 and in_distance_range_pose(np.zeros(3), center, d_min, d_max):
-                    track_label_uuid = gt_data[i]["track_label_uuid"]
+                    if gt_data[i]["track_label_uuid"] not in track_label_uuids:
```

Comment:
Hi Anna, could you add a brief inline comment here explaining the logic for what we are doing here?